libs/core-ui/src/lib/Interfaces/IFairnessData.ts

```diff
@@ -19,12 +19,15 @@ export type PredictionType =
 export interface IMetricResponse {
   global: number;
   bins: number[];
+  bounds?: number[];
```

Comment:
Should I structure it as a dict? I think an object might be better, but I'm not sure how to pass that through the Python flask response

Reply:
The individual pieces of the object can be passed individually and then you can reassemble it on the other end. Or it might even work to have it as JSON.